modules/mco-update-boot-images-disable.adoc

```diff
@@ -23,6 +23,8 @@ $ oc -n openshift-config patch cm admin-acks --patch '{"data":{"ack-4.18-boot-im
 ----
 
 * If you do not want the updated boot image feature enabled, explicitly disable the feature by using the following procedure.
+
+It is important to note that if you use boot images from the AWS Marketplace or the GCP Marketplace, enabling this feature overwrites those images with a standard {op-system-first} image. You should explicitly disable this feature and not patch the `admin-acks` config map. If you accidentally enable the updated boot image feature, you can disable the feature by using the following procedure. Then, replace the marketplace boot images by creating compute machine sets, as described in _Creating a compute machine set on GCP_ or _Creating a compute machine set on GCP_.
```

Comment:
@zaneb caught that instead of creating new machinesets you should modify the existing one e.g the final header in https://access.redhat.com/articles/6675791